docs: Update docs w.r.t. new structure (#490)
ansys/pyfluent-visualization: doc/source/users_guide/index.rst

==========
User guide
==========

> PyFluent-Visualization enables postprocessing of Fluent results,

Reviewer: Post-processing is supposed to have an hyphen, I think
```suggestion
PyFluent-Visualization enables post-processing of Fluent results,
```

Graphics operations
-------------------
Examples of graphics operations that PyFluent-Visualization supports:

> Launching Fluent and setting up data

Reviewer: ```suggestion
Launch Fluent and read data
```

~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Example launching Fluent, and reading case and data files:

Reviewer 2: @prmukherj 
- Where we previously had sentences, we now have fragments (lines 11 and 15). We can't have these changes.
- The change on line 15 suggests that we have an example that's only about launching Fluent and reading files.
- The flow is all wrong. E.g., "Launch Fluent and read data" directly follows "Examples of graphics operations that PyFluent-Visualization supports:". That just doesn't flow and gives the reader a poor perspective on what to expect.

Reviewer: In my understanding, this is just one subsection, line 15 only refers to this subsection. For example, looking at the table of contents of this file:
![image](https://github.com/user-attachments/assets/39b3a34c-889d-4cbb-8102-96b075a2eba6)

With that said, I agree lines 11 and 15 should probably just be removed at this point, they aren't really adding useful information. We already have on lines 6-7 a brief introduction that mentions PyFluent-Visualization.

> You launch Fluent, read a case and hybrid initialize to get the data:

Reviewer: ```suggestion
Example launching Fluent, and reading case and data files:
```
This example is not using hybrid initialization as far as I can tell

    solver_session.tui.file.read_case(import_case)
    solver_session.tui.file.read_data(import_data)

Graphics operations
-------------------

> Graphics operations that PyFluent-Visualization supports:

Reviewer 3: ```suggestion
Use PyFluent-Visualization graphics operations as shown in following code examples.
```
The sentence wasn't a complete thought as originally written.

        "solid_up:1:830",
        "solid_up:1:830-shadow",
    ]

    mesh_object_1 = Mesh(solver=solver_session, show_edges=True, surfaces=mesh_surfaces_list)
    mesh_window = GraphicsWindow(grid=(1, 2))
    mesh_window.add_graphics(mesh1, position=(0, 0))

    mesh_object_2 = Mesh(solver=solver_session, surfaces=mesh_surfaces_list)
    mesh_object_2.show_edges = False

    mesh_window.add_graphics(mesh2, position=(0, 1))
    mesh_window.show()

> Display plane-surface

Reviewer 3: I realize plane-surface is what is used in the code. However, wouldn't you use "plane surface" in a heading or sentence?

    cont_surfaces_list = [
        "in1",
        "in2",
        "in3",
        "out1",
        "solid_up:1",
        "solid_up:1:830",
    ]
    temperature_contour_manifold = Contour(
        solver=solver_session,
        field="temperature",
        surfaces=cont_surfaces_list,
    )
    contour_window = GraphicsWindow()
    contour_window.add_graphics(temperature_contour_manifold)
    contour_window.show()

Display vector
~~~~~~~~~~~~~~

.. code-block:: python

    from ansys.fluent.visualization import Vector

    velocity_vector = Vector(
        solver=solver_session,
        field="pressure",
        surfaces=["solid_up:1:830"],
        scale=2,
    )
    vector_window = GraphicsWindow()
    vector_window.add_graphics(velocity_vector)
    vector_window.show()

Display pathlines
~~~~~~~~~~~~~~~~~

.. code-block:: python

    from ansys.fluent.visualization import Pathline

    pathlines = Pathline(solver=solver_session)
    pathlines.field = "velocity-magnitude"
    pathlines.surfaces = ["inlet", "inlet1", "inlet2"]

    pathlines_window = GraphicsWindow()
    pathlines_window.add_graphics(pathlines)
    pathlines_window.show()

Plot operations
---------------

> Plot operations that PyFluent-Visualization supports:

Reviewer 3: ```suggestion
Use PyFluent-Visualization plot operations as shown in following code examples.
```
Again, the introduction to the subsequent sections needs to be a complete sentence.

        solver=solver_session,
        surfaces=["outlet"],
        y_axis_function="temperature",
    )
    xy_plot_window = GraphicsWindow()
    xy_plot_window.add_graphics(xy_plot)
    xy_plot_window.show()

Display solution residual plot
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

.. code-block:: python

    from ansys.fluent.visualization import Monitor

    residual = Monitor(solver=solver_session)
    residual.monitor_set_name = "residual"
    monitor_window = GraphicsWindow()
    monitor_window.add_graphics(residual)
    monitor_window.show()

Display solution monitors plot
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

.. code-block:: python

    solver_session.settings.solution.initialization.hybrid_initialize()
    solver_session.settings.solution.run_calculation.iterate(iter_count=50)

    mass_bal_rplot = Monitor(solver=solver_session)
    mass_bal_rplot.monitor_set_name = "mass-bal-rplot"
    monitor_window = GraphicsWindow(grid=(1, 2))
    monitor_window.add_graphics(mass_bal_rplot, position=(0, 0))

    point_vel_rplot = Monitor(solver=solver_session, monitor_set_name="point-vel-rplot")
    monitor_window.add_graphics(point_vel_rplot, position=(0, 1))
    monitor_window.show()

> Interactive Graphics

Reviewer 3: ```suggestion
Interactive graphics
```
Use sentence case titles.

--------------------
The ``GraphicsWindow`` class provides for managing and directly interacting

> with the graphics window. By registering the window with ``EventsManager``, you

Reviewer 3: I think EventsManager is a proper noun but not a code entity. If it appears in a UI as the name of a component, use bold formatting. Otherwise, no formatting is necessary.

can update graphics during run time and create animations.

> The following example updates the graphics window (contour_window, xy_plot_window and

Reviewer 3: ```suggestion
The following example updates the graphics window (contour_window, xy_plot_window, and
```
Serial (or what is also known as Oxford comma). Always use a comma before the "and" or "or" in a list of three or more items.

Reviewer 2: @PipKat @prmukherj @mkundu1 I believe that we need to change our config to enforce this style.

> monitor_window) during solution initialization and whenever data is read.

Reviewer 3: ```suggestion
``monitor_window``) during solution initialization and whenever data is read.
```

> During the calculation, it also updates the graphics window at
> the end of every time step. These are done via. explicit callback registrations.

Reviewer 3: ```suggestion
During the calculation, the ``GraphicsWindow`` class also uses explicit callback
registrations to update the graphics window at the end of every time step.
```
I'm assuming "it" refers to the aforementioned class. The pronoun "it" is too far from whatever noun it is referring to, which is why I made the change I did. I only hope it is correct.

        "solid_up:1:830",
        "solid_up:1:830-shadow",
    ]

    mesh1 = Mesh(solver=solver_session, show_edges=True, surfaces=mesh_surfaces_list)

>     p1 = GraphicsWindow(grid=(1, 2))

Reviewer 2: @prmukherj I still haven't managed to figure out what a p is.

Reviewer: I am thinking it is probably "p" for "plot"

    p1.add_graphics(mesh1, position=(0, 0))

    mesh2 = Mesh(solver=solver_session, surfaces=mesh_surfaces_list)
    mesh2.show_edges = False

    p1.add_graphics(mesh2, position=(0, 1))
    p1.show()

Display plane-surface
~~~~~~~~~~~~~~~~~~~~~

.. code:: python

    from ansys.fluent.visualization import Surface

    surf_xy_plane = Surface(solver=solver_session)
    surf_xy_plane.definition.type = "plane-surface"
    surf_xy_plane.definition.plane_surface.creation_method = "xy-plane"
    plane_surface_xy = surf_xy_plane.definition.plane_surface.xy_plane

>     plane_surface_xy.z = -0.0441921

Reviewer 2: @prmukherj I have overlooked classes like this one in the refactoring. This awkward construction sequence is inconsistent with what we achieved for the graphical classes.

Author: Yes, totally agreed. Will encapsulate it today.

    iso_surf1 = surf_outlet_plane.definition.iso_surface
    iso_surf1.field = "y-coordinate"
    iso_surf1.iso_value = -0.125017
    p3 = GraphicsWindow()
    p3.add_graphics(surf_outlet_plane)
    p3.show()

Display contour
~~~~~~~~~~~~~~~
This example shows how you can display a contour:

.. code:: python

    from ansys.fluent.visualization import Contour

    cont_surfaces_list = [
        "in1",
        "in2",
        "in3",
        "out1",
        "solid_up:1",
        "solid_up:1:830",
    ]
    temperature_contour_manifold = Contour(
        solver=solver_session,
        field="temperature",
        surfaces=cont_surfaces_list,
    )
    p4 = GraphicsWindow()
    p4.add_graphics(temperature_contour_manifold)
    p4.show()

Display vector
~~~~~~~~~~~~~~
This example shows how you can display a vector:

.. code:: python

    from ansys.fluent.visualization import Vector

    velocity_vector = Vector(
        solver=solver_session,
        field="pressure",
        surfaces=["solid_up:1:830"],
        scale=2,
    )
    p5 = GraphicsWindow()
    p5.add_graphics(velocity_vector)
    p5.show()

Display pathlines
~~~~~~~~~~~~~~~~~

> This example shows how you can display a pathlines:

Reviewer: ```suggestion
This example shows how you can display pathlines:
```
